code/modules/events/holiday/vday.dm

```diff
@@ -116,7 +116,15 @@
 	               "Roses are red, text is green, I love you more than cleanbots clean.",
 	               "If you were a carp I'd fi-lay you.",
 	               "I'm a nuke op, and my pinpointer leads to your heart.",
-	               "Wanna slay my megafauna?" )
+	               "Wanna slay my megafauna?",
+	               "I'm a clockwork cultist. Or zl inyragvar.",
+	               "If you were a disposal bin I'd ride you all day.",
+	               "Put on your explorer's suit because I'm taking you to LOVEaland.",
```

Comment:
how does love-a-land not roll off the tongue

guess you're not used to using yours